webpack.config.js

```diff
@@ -222,17 +249,20 @@ if (ENV === 'production') {
           use: 'babel-loader',
         },
         {
-          test: /\.scss$/,
+          test: /\.s[ac]ss$/i,
```

Comment:
are you using both scss and sass extensions in here?  Is there something that's preventing you from only using one of these?  It seems like a strange decision.

Reply:
This was left over from my attempts to resolve the node-sass issues. Regardless, why restrict which file extension when one is synonymous to the other?

Reply:
It depends on who's looking at it and why they are - PRs are weird in that you review them with the assumption that every choice that is made is a conscious choice.  A change like this implies to reviewer that SASS files were added in addition to the SCSS files, which, if true, would have been a very strange choice to make.